### PR TITLE
Update ipsec-road.rst

### PR DESCRIPTION
**Pull request by mimugmail**
Added a note after save. Make a difference when IPsec was started before or not.

### CODE REVIEW

File: source/manual/how-tos/ipsec-road.rst [hunk 1]
```diff
@@ -238,6 +238,12 @@ And Apply changes:
 
 -----------------------------
 
+.. Note::
+
+   If you already had IPsec enabled and added Road Warrior setup, it's important to 
+   restart the whole service via **System->Diagnostics->Services->Strogswan** since applying
```
Comment (fichtner) [inline source/manual/how-tos/ipsec-road.rst]: Doesn't the service widget do this on all the IPsec pages as well?

Comment (fichtner) [reply source/manual/how-tos/ipsec-road.rst]: But the reload button would so the same thing on dashboard, diagnostics and page control unless there is a bug...?

Comment (mimugmail) [reply source/manual/how-tos/ipsec-road.rst]: It needs a restart, not a relaod. A reload is done on every change. The IPSec pages don't have an own services widget. And the Services widget via Dashboard can also be removed by the user, that's why is I have choosen the most stable approach.

Comment (fichtner) [reply source/manual/how-tos/ipsec-road.rst]: Err, lol, I broke it in 2016 and nobody noticed. :) https://github.com/opnsense/core/commit/e99ae7ad17b

Comment (mimugmail) [reply source/manual/how-tos/ipsec-road.rst]: Dont hate the Player :) I'll change it tomorrow .. gn

Comment (fichtner) [reply source/manual/how-tos/ipsec-road.rst]: No, no, thanks for finding this! gn

**Comment by fichtner** [comment]
Merged, thanks. I'll add the service control to 18.7.6.

**Comment by mimugmail** [comment]
Thx, I'll rewrite the road warrior part today fitting more clients with the new multiselect for hash and DH

**Comment by fichtner** [comment]
great, looking forward to merging it ❤️

**Pull request merged**